### ISSUE TITLE
Desired State Configureation WindowsFeature Name Field Intellisense Crashes PowerShell Session

### ISSUE THREAD

**Issue by Persistent13** [PowerShell/vscode-powershell#391]
### System Details

- Operating system name and version: Microsoft Windows 10 Pro Insider Preview
- VS Code version: 1.8.0
- PowerShell extension version: 0.8.0
- Output from `$PSVersionTable`: 

```
Name                           Value
----                           -----
PSVersion                      5.1.14971.1000
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.14971.1000
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

### Issue Description

When writing a DSC file if I attempt to use intellisense to complete the name field of WindowsFeature shown below the PowerShell session will crash.

```
configuration WebServer
{
    node $ComputerName
    {
        File DSC-Log
        {
            Ensure = 'Present'
            Type = 'Directory'
            DestinationPath = "$env:SystemDrive\DSCBuildLog"
        }
        WindowsFeature IIS
        {
            Ensure = 'Present'
            LogPath = "$env:SystemDrive\DSCBuildLog\IIS.log"
            Name = 
        }
    }
}
```

The crashing stops if I remove the DSC-Log code block.

Let me know if you need any further details!

### Attached Logs

[Logs](https://cns-my.sharepoint.com/personal/dclark_cns-service_com/_layouts/15/guestaccess.aspx?guestaccesstoken=zSDa5o%2bOU2dBjcif5RExheKaWegUeslG18RNaOvd4Vo%3d&docid=0fbc13279ac904e29ab4566b2b92dcf5b&rev=1&expiration=2017-01-15T23%3a19%3a20.000Z)

**Comment by daviwil** [comment]
Thanks Dakota!  I see the crash in the logs, will try to get this fixed for an 0.8.1 update next week.

**Comment by daviwil** [comment]
I've got a fix ready for this, it will go out with the 0.8.1 update this week!
